src/framework/standard/command.rs

```diff
@@ -90,6 +93,13 @@ pub struct Command {
     pub(crate) aliases: Vec<String>,
 }
 
+impl fmt::Debug for Command {
+    // add checks somehow?
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "Point {{ bucket: {:?}, desc: {:?}, example: {:?}, usage: {:?}, min_args: {:?}, max_args: {:?}, required_permissions: {:?}, allowed_roles: {:?}, help_available: {:?}, dm_only: {:?}, guild_only: {:?}, owners_only: {:?}}}", self.bucket, self.desc, self.example, self.usage, self.min_args, self.max_args, self.required_permissions, self.allowed_roles, self.help_available, self.dm_only, self.guild_only, self.owners_only)
```

Comment:
Please use [debug_struct](https://doc.rust-lang.org/std/fmt/struct.Formatter.html#method.debug_struct) instead of this, as it will not only help with making this more concise to read, it'll also make `Command` pretty-printable.